howso/utilities/feature_attributes/pandas.py

```diff
@@ -314,7 +317,11 @@ def _as_float(value, dtype):
                     min_time = time_to_seconds(time_column.min().time())
                     max_time = time_to_seconds(time_column.max().time())
 
-                    return {'min': min_time, 'max': max_time, 'allow_null': allow_null}
+                    return {
+                        'min': min_time, 'max': max_time,
+                        'actual_min': min_time, 'actual_max': max_time,
```

Comment:
Is there a related howso-engine change for the addition of these keys?

Reply:
https://github.com/howsoai/howso-engine/pull/415